command/helpers.go

```diff
@@ -75,6 +75,11 @@ func formatTimeDifference(first, second time.Time, d time.Duration) string {
 	return second.Truncate(d).Sub(first.Truncate(d)).String()
 }
 
+// formatTimePretty rounds off time difference to the nearest second for nicer display
+func formatTimePretty(first, second time.Time) string {
```

Comment:
Can we use something like: https://godoc.org/github.com/dustin/go-humanize#Time

The output of this still could become huge if the allocation is many days old.

Reply:
You may want to specialize the 0 case since older allocations won't have the modify time and when it gets displayed it would show something like 40 years ago!

Reply:
LOL, good way to remind people that for engineers, time doesn't exist prior to Jan 1 1970. Will fix